### PR TITLE
Add node to nix

### PR DESCRIPTION
**Pull request by BrianHicks**
add some stuff I was missing from nix!

### CODE REVIEW

**Comment by omnibs** [comment]
Came here to test it on AV's request, seems to work:

```
$ which node
/nix/store/b7bv1hbj8c51a98mcvjk6m53bg6xyc2p-nodejs-8.11.4/bin/node
$ node
> console.log('hi!')
hi!
```

File: .envrc.nix [hunk 1]
```diff
@@ -1 +1,2 @@
 use nix
+export PATH="./node_modules/.bin:$PATH"
```
Comment (avh4) [inline .envrc.nix]: is this preferred over just using `npx <thing>` to run thing?

Comment (allenap) [reply .envrc.nix]: ☁️ Make it an absolute path?

Comment (BrianHicks) [reply .envrc.nix]: @allenap ahh, good catch!

@avh4 I prefer it so that I can just run things "normally"

**Pull request merged**